apps/console/src/features/groups/components/edit-group/edit-group-roles.tsx

```diff
@@ -170,10 +175,17 @@ export const GroupRolesList: FunctionComponent<GroupRolesPropsInterface> = (
     }, [ group ]);
 
     useEffect(() => {
-        getRolesList(null)
-            .then((response) => {
-                setPrimaryRoles(response.data.Resources);
-            });
+        if (isRootOrganization) {
+            getRolesList(null)
```

Comment:
Might be better to include a statement if the API call resulted an error as well. Check in other places too

Reply:
Done in https://github.com/wso2/identity-apps/pull/3617/commits/630fa8dda5d13075f2d424b2493249572e255217